src/net/sourceforge/kolmafia/textui/Line.java

```diff
@@ -132,13 +134,12 @@ public String toString() {
     return this.content;
   }
 
-  public class Token {
-    final int offset;
+  public class Token extends Range {
     final String content;
     final String followingWhitespace;
     final int restOfLineStart;
```

Comment:
Do we still want / need restOfLineStart? It's basically equivalent to this.getEnd().getCharacter() + followingWhitespace.length(), right?

Reply:
For the final token, its value doesn't match its content.

Reply:
Can you elaborate? I don't know what you mean by that.

Reply:
Erm... apologies, that wasn't even... What I meant wasn't even correct...


This is still a handy shortcut to keep, as it's used a few times (quite a few times in the tests), and it would take space to have to calculate it each time.
It's not "essential", just "convenient". Your call, I guess.